### ISSUE TITLE
Gnome 43 Runtime is End of Life

### ISSUE THREAD

**Issue by thatLeaflet** [FineFindus/eyedropper#85]
**Describe the bug**
Eyedropper uses the Gnome 43 flatpak runtime, which is now EOL.

```
Info: runtime org.gnome.Platform branch 43 is end-of-life, with reason:
   The GNOME 43 runtime is no longer supported as of September 20, 2023. Please ask your application developer to migrate to a supported platform.
Info: applications using this runtime:
   com.github.finefindus.eyedropper
```

**Comment by FineFindus** [comment]
Thanks for the info, a new release with the new runtime is planned for next week. If you can't wait, there is already a beta out.
Will keep this open until the new version is released.

**Comment by FineFindus** [comment]
Fixed in 0658771fd44c180299e8431610f385a1e5d65c63.
